packages/design-tokens/scripts/style-dictionary.js

```diff
@@ -0,0 +1,495 @@
+        files: [
+          {
+            filter: token => token.filePath.includes('viewport'),
+            format: 'custom/format/custom-media',
+            destination: 'viewport.css',
+          },
+        ],
+      },
+      scss: {
+        buildPath: `${outputPath}/scss/`,
+        transformGroup: 'scss',
+        // map the array of token file paths to style dictionary output files
+        files: files.map(filePath => {
+          return {
+            format: `scss/variables`,
+            destination: filePath.replace(`.json`, `.scss`),
+            filter: token => token.filePath === filePath,
+            options: {
+              outputReferences: true,
+            },
+          }
+        }),
+      },
+      js: {
+        buildPath: `${outputPath}/js/`,
+        transforms: ['name/js/es6', 'pxToRem'],
+        // map the array of token file paths to style dictionary output files
+        files: files.map(filePath => {
+          return {
+            format: `javascript/es6`,
+            destination: filePath.replace(`.json`, `.js`),
+            filter: token => token.filePath === filePath,
+          }
+        }),
+      },
+      jsModule: {
+        buildPath: `${outputPath}/js/module/`,
+        transforms: ['pxToRem'],
+        // map the array of token file paths to style dictionary output files
+        files: files.map(filePath => {
+          return {
+            format: `javascript/module`,
+            destination: filePath.replace(`.json`, `.js`),
+            filter: token => token.filePath === filePath,
+          }
+        }),
+      },
+      tsTypes: {
+        buildPath: `${outputPath}/ts/`,
+        transforms: ['pxToRem'],
+        // map the array of token file paths to style dictionary output files
+        files: files.map(filePath => {
+          return {
+            format: `typescript/module-declarations-v2`,
+            destination: filePath.replace(`.json`, `.d.ts`),
+            filter: token => token.filePath === filePath,
+          }
+        }),
+      },
+      ts: {
+        buildPath: `${outputPath}/ts/`,
+        transforms: ['pxToRem'],
+        // map the array of token file paths to style dictionary output files
+        files: files.map(filePath => {
+          return {
+            format: `javascript/module-v2`,
+            destination: filePath.replace(`.json`, `.js`),
+            filter: token => token.filePath === filePath,
+          }
+        }),
+      },
+    }
+
+    const config = {
+      ...{include: include ? [include] : undefined},
+      source: files,
+      ...customParseConfig,
+      platforms: platforms ? platforms : defaultPlatformConfig,
+    }
+    return config
+  }
+
+  //build all tokens
+  _StyleDictionary
+    .extend({
+      ...getConfig(glob.sync([...source])),
+    })
+    .buildAllPlatforms()
+}
+
+module.exports = {
+  buildPrimitives,
+  StyleDictionary,
+  pathToKebabCase,
+  pathToDotNotation,
+  capitalize,
+  pathToPascalCase,
```

Comment:
Do you need to export them?

Reply:
Probably not, let me remove these now 👍 thanks